src/System.Runtime/tests/System/ActivatorTests.cs

```diff
@@ -392,5 +396,158 @@ static void TestingActivationAttributes1()
             Activator.CreateInstance(typeof(ClassWithIsTestedAttribute), null, new object[] { });
 
         }
+
+        [Theory]
+        [MemberData(nameof(TestingCreateInstanceFromObjectHandle))]        
+        static void TestingCreateInstanceFromObjectHandle(string physicalFileName, string assemblyFile, string type, string returnedFullNameType, Type exceptionType)
+        {
+            ObjectHandle oh = null;
+
+            if (exceptionType != null)
+            {
+                Assert.Throws(exceptionType, () => Activator.CreateInstanceFrom(assemblyFile: assemblyFile, typeName: type));
+            }
+            else
+            {
+                oh = Activator.CreateInstanceFrom(assemblyFile: assemblyFile, typeName: type);
+                Assert.NotNull(oh);
+                Assert.Equal(returnedFullNameType, oh.Unwrap().GetType().FullName);
+            }
+
+            if (exceptionType != null)
+            {
+                Assert.Throws(exceptionType, () => Activator.CreateInstanceFrom(assemblyFile: assemblyFile, typeName: type, null));
+            }
+            else
+            {
+                oh = Activator.CreateInstanceFrom(assemblyFile: assemblyFile, typeName: type, null);
+                Assert.NotNull(oh);
+                Assert.Equal(returnedFullNameType, oh.Unwrap().GetType().FullName);
+            }
+            Assert.True(File.Exists(physicalFileName));
+        }
+
+        public static IEnumerable<object[]> TestingCreateInstanceFromObjectHandle()
+        {
+            //string physicalFileName, string assemblyFile, string typeName, returnedFullNameType, expectedException
+            yield return new object[] { "TestLoadAssembly.dll", "TestLoadAssembly.dll", "PublicClassSample", "PublicClassSample", null };
+            yield return new object[] { "TestLoadAssembly.dll", "testloadassembly.dll", "publicclasssample", "PublicClassSample", typeof(TypeLoadException) };
+
+            yield return new object[] { "TestLoadAssembly.dll", "TestLoadAssembly.dll", "PrivateClassSample", "PrivateClassSample", null };
+            yield return new object[] { "TestLoadAssembly.dll", "testloadassembly.dll", "privateclasssample", "PrivateClassSample", typeof(TypeLoadException) };
+
+            yield return new object[] { "TestLoadAssembly.dll", "TestLoadAssembly.dll", "PublicClassNoDefaultConstructorSample", "PublicClassNoDefaultConstructorSample", typeof(MissingMethodException) };
+            yield return new object[] { "TestLoadAssembly.dll", "testloadassembly.dll", "publicclassnodefaultconstructorsample", "PublicClassNoDefaultConstructorSample", typeof(TypeLoadException) };
+        }
+
+        [Theory]
+        [MemberData(nameof(TestingCreateInstanceObjectHandle))]
+        static void TestingCreateInstanceObjectHandle(string assemblyName, string type, string returnedFullNameType, Type exceptionType)
+        {
+            ObjectHandle oh = null;
+                
+            if(exceptionType != null)
```

Comment:
Nit: Space `if (`

Reply:
done

Reply:
i'm suspecting that some of you are bot, should be night in Redmond :smiley: